testing/src/test/java/org/oppia/android/testing/CoroutineExecutorServiceTest.kt

```diff
@@ -72,12 +69,16 @@ class CoroutineExecutorServiceTest {
   @field:BackgroundDispatcher
   lateinit var backgroundCoroutineDispatcher: CoroutineDispatcher
 
-  @Inject lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
+  @Inject
+  lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
```

Comment:
Did ktlint do this? Both are stylistically correct per the Kotlin style guide, so ktlint might be a bit opinionated here (which is fine).

Reply:
Actually I had reformatted the file which made this change.